`treehouses shutdown` (fixes #992)

Neethu-Mohan opened the pull request:
treehouses shutdown.sh included

LordJashin32 commented:
Is there any reason we call this `shutdowns` instead of `shutdown`. /usr/bin/shutdown and `shutdown` is indeed a function already available in the shell. But that doesn't mean we need to avoid the naming. When i tried `./cli.sh shutdown` it still worked for me. Is there something i miss?

Neethu-Mohan commented:
@LordJashin32, yes, you are right. We don't need to change the name from 'shutdown' to 'shutdowns'. Previously, when I tried to use "shutdown/shutdown now" command in a function called "shutdown", it didn't work for me. That is why I renamed the file to shutdowns.sh. Now, I resolved it and updated the file. Please test and review the same
